lib/patch/mutation-observer.js

```diff
@@ -39,7 +39,7 @@ function patchClass(className) {
   var prop;
   for (prop in instance) {
     (function (prop) {
-      if (typeof global[className].prototype !== undefined) {
+      if (typeof global[className].prototype !== 'undefined') {
```

Comment:
ah...how to build a test spec for this situation? I'm afraid that I have no experience to build such a test spec...